lessons/8_loops/index.html

```diff
@@ -0,0 +1,308 @@
+                for (let i = 0; i < 20; i++) {
+                    console.log(i + 1)
+                }
+            </code></pre>
+            <br/>
+            <div class="fragment">
+                <h4>Print list of items</h4>
+                <pre><code data-trim class="hljs">
+                    const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                    for (let i = 0; i < items.length; i++) {
+                        console.log(items[i]);
+                    }
+                </code></pre>
+            </div>
+       </section>
+
+       <section>
+           <h4>Print students' full name.</h4>
+           <pre><code data-trim class="hljs">
+                const students = [{
+                    name: "Anna",
+                    lastname: "Danielkiewicz"
+                },{
+                    name: "Sergei",
+                    lastname: "Golovnev"
+                },{
+                    name: "Natalia",
+                    lastname: "Gubanova"
+                },{
+                    name: "Sadat",
+                    lastname: "Rashid"
+                }];
+
+                for (let i = 0; i < students.length; i++) {
+                    console.log(`Student ${i + 1}: ${students[i].name} ${students[i].lastname}`)
+                }
+            </code></pre>
+       </section>
+
+       <section>
+           <h4>Render 4 buttons in your screen</h4>
+           [1] [2] [3] [4]
+           <pre><code data-trim class="">
+                &lt;!DOCTYPE html&gt;
+                &lt;html&gt;
+                    &lt;head&gt;
+                        &lt;meta charset="UTF-8"&gt;
+                        &lt;title&gt;Button&lt;/title&gt;
+                    &lt;/head&gt;
+                
+                    &lt;body&gt;
+                        &lt;p&gt;Buttons&lt;/p&gt;
+                        &lt;div id="container"&gt;&lt;/div&gt;
+
+                        &lt;script&gt;
+                            const buttonsContainer = document.getElementById('container');
+                            for(let i=0; i < 4; i++) {
+                                const buttonElement = document.createElement('button');
+                                buttonElement.innerText = i + 1;
+                                buttonsContainer.appendChild(buttonElement)
+                            }
+                        &lt;/script&gt;
+                    &lt;/body&gt;
+                &lt;/html&gt;
+           </code></pre>
+       </section>
+
+       <section>
+           <h3>Array Iteration Methods</h3>
+           <ul>
+                <li>Array.forEach()</li>
+                <pre><code data-trim class="hljs">
+                    const numbers = [45, 4, 9, 16, 25];
+                    numbers.forEach(function(number) {
+                        console.log(number);
+                    });
+                </code></pre>
+            <li>Array.map()</li>
```

Comment:
Would you explain what they do or just mention it and then they free to read more on MDN?

Reply:
I am going to explain a bit what they do and then they can read more on MDN.